extensions/adobe/experience/adcloud/searchads/campaign.schema.json

```diff
@@ -0,0 +1,166 @@
+  "definitions": {
+    "searchads-campaign": {
+      "properties": {
+        "searchads:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the search advertising platform."
+        },
+        "searchads:campaignName": {
+          "title": "Campaign Name",
+          "type": "string",
+          "description": "Name of the Campaign."
+        },
+        "searchads:campaignDescription": {
+          "title": "Description",
+          "type": "string",
+          "description": "Description of the Campaign."
+        },
+        "searchads:campaignStatus": {
+          "title": "Campaign Status",
+          "type": "string",
+          "description": "Campaign Status indicates if underlying ads are eligible to serve.",
+          "enum": [
+            "Active",
+            "Paused",
+            "Deleted",
+            "Ended",
+            "Pending",
+            "Inactive",
+            "Disapproved",
+            "Incomplete",
+            "Offline",
+            "On Hold (Editing)",
+            "On Hold (Other)",
+            "Duplicate",
+            "Orphan",
+            "Optimize",
+            "Disabled",
+            "Unknown",
+            "Dummy"
+          ],
+          "meta:enum": {
+            "Active": "Active",
+            "Paused": "Paused",
+            "Deleted": "Deleted",
+            "Ended": "Ended",
+            "Pending": "Pending",
+            "Inactive": "Inactive",
+            "Disapproved": "Disapproved",
+            "Incomplete": "Incomplete",
+            "Offline": "Offline",
+            "On Hold (Editing)": "On Hold (Editing)",
+            "On Hold (Other)": "On Hold (Other)",
+            "Duplicate": "Duplicate",
+            "Orphan": "Orphan",
+            "Optimize": "Optimize",
+            "Disabled": "Disabled",
+            "Unknown": "Unknown",
+            "Dummy": "Dummy"
+          }
+        },
+        "searchads:channelType": {
+          "title": "Channel Type",
+          "type": "string",
+          "description": "Channel Type for the Campaign"
+        },
+        "searchads:budget": {
+          "title": "Budget",
+          "type": "number",
+          "description": "Budget Allocated for the Campaign"
+        },
+        "searchads:budgetType": {
+          "title": "Budget Type",
+          "type": "string",
+          "description": "Budget Type indicates how the allocated budget will be spent",
+          "enum": [
+            "Daily",
+            "Monthly",
+            "Lifetime",
+            "Daily Spend Until Depleted",
+            "Day of Week"
+          ],
+          "meta:enum": {
+            "daily": "Daily",
+            "Monthly": "Monthly",
+            "Lifetime": "Lifetime",
+            "Daily Spend Until Depleted": "Daily spend until depleted",
+            "Day of week": "Day of week"
+          }
+        },
+        "searchads:adDeliveryType": {
```

Comment:
If there are only two states of adDelivery, should this be boolean?

Reply:
Boolean doesn't signify anything. I would rather put it as a string. For now enum should suffice.